tests/ui/tuple/add-tuple-within-arguments.stderr

```diff
@@ -8,7 +8,7 @@ note: function defined here
   --> $DIR/add-tuple-within-arguments.rs:1:4
    |
 LL | fn foo(s: &str, a: (i32, i32), s2: &str) {}
```

Comment:
These spans don't seem correct

Reply:
Agreed, there's a few places these need fixing before this gets merged.